static/app/types/hooks.tsx

```diff
@@ -203,6 +210,13 @@ export type SettingsHooks = {
   'settings:organization-navigation-config': SettingsConfigHook;
 };
 
+/**
+ * Callback hooks
```

Comment:
Can you expand on the description a little bit actually

Reply:
@evanpurkhiser Specifically for the `callback:on-guide-update` hook?

Reply:
nah, what this callback hooks category might be fore basically